### ISSUE TITLE
Error while using api.update_status('hello')

### ISSUE THREAD

**Issue by harshdattani** [tweepy/tweepy#659]
Getting this error while running api.update_status('hello')
Library is imported perfectly and all keys are verified to be truely entered.

Error:
Traceback (most recent call last):
  File "runme.py", line 12, in <module>
    api.update_status('tweepy + oauth!')
  File "/usr/local/lib/python2.7/dist-packages/tweepy/api.py", line 193, in update_status
    )(post_data=post_data, *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/tweepy/binder.py", line 243, in _call
    return method.execute()
  File "/usr/local/lib/python2.7/dist-packages/tweepy/binder.py", line 189, in execute
    raise TweepError('Failed to send request: %s' % e)
tweepy.error.TweepError: Failed to send request: cannot make memory view because object does not have the buffer interface

**Comment by fitnr** [comment]
I believe this is a bug in requests, a library tweepy relies on. I had the same issue and solved it by downgrading to requests 2.7.0: 
```
pip uninstall requests
pip install requests==2.7
```

**Comment by yprez** [comment]
I think it's the same issue as #66

**Comment by yprez** [comment]
I think it's the same issue as #644 #578 #554 , etc.

tl;dr - try using: `api.update_status(status='tweepy + oauth!')`

**Comment by jontsai** [comment]
I'm having a similar issue:
`api.lookup_users(user_ids=chunk)` results in `Failed to send request: cannot make memory view because object does not have the buffer interface`

Downgrading `requests==2.8.1` to `requests==2.7.0` also seems to fix it for me.

**Comment by pcawthron** [comment]
I'm very new to tweepy but api.update_status("A test from the tweepy API") works fine here with requests 2.9.1 with Python 2.7.11 32-bit on Windows 8.1 Pro 64-bit.

My packages are:

PySide 1.2.4 has no metadata
Pygments 2.0.2 has no metadata
cycler 0.9.0 has no metadata
decorator 4.0.6 has no metadata
ipykernel 4.2.1 has no metadata
ipython-genutils 0.1.0 has no metadata
ipython 4.0.1 has no metadata
jupyter-client 4.1.1 has no metadata
jupyter-core 4.0.6 has no metadata
matplotlib 1.5.0 has no metadata
numexpr 2.4.6 has no metadata
numpy 1.9.3 has no metadata
oauthlib        - 1.0.3        - active 
pandas 0.17.1 has no metadata
path.py 8.1.2 has no metadata
pickleshare     - 0.5          - active 
pip 7.1.2 has no metadata
pyparsing 2.0.6 has no metadata
python-dateutil 2.4.2 has no metadata
pytz 2015.7 has no metadata
pyzmq 15.1.0 has no metadata
qtconsole 4.1.1 has no metadata
requests-oauthlib 0.6.0 has no metadata
requests 2.9.1 has no metadata
setuptools 18.2 has no metadata
simplegeneric   - 0.8.1        - active 
six 1.10.0 has no metadata
tables 3.2.2 has no metadata
traitlets 4.0.0 has no metadata
tweepy 3.5.0 has no metadata
twitter 1.17.1 has no metadata
twython 3.3.0 has no metadata
wheel 0.26.0 has no metadata
yolk            - 0.4.3        - active

**Comment by augusto2012** [comment]
Downgrading the module "requests" 2.7 did for me, thank you
